addons/mass_mailing_sms/views/mailing_mailing_views.xml

```diff
@@ -268,16 +268,16 @@
           </p>
         </field>
     </record>
-    <record id="mailing_mailing_action_sms_view_kanban" model="ir.actions.act_window.view">
+    <record id="mailing_mailing_action_sms_view_tree" model="ir.actions.act_window.view">
```

Comment:
Is the order of the views done by sequence number in the file ? Or is it by alphabetical order ?
If it's alphabetical then no need to change the records, just the sequence should be ok.

Reply:
Hello @thjo-odoo :slightly_smiling_face: 

I think it is done with the sequence number, this is why I exchanged the numbers.
However, I found it weird to have a view with sequence number 2 above a view with sequence number 1 so I also exchanged their order in the file since they weren't so big. It isn't necessary just a bit more logical in my sens. :slightly_smiling_face:  

It adds a little bit of diff but keep things in order. However, if it is too bothering I can just reset to the basic record order (kanban view and then tree one). What do you think?

Thank you!

Reply:
Yeah you're right it would be weird to have the record 1 below the record 2 let's keep it this way then.